cmd/crossbuild.go

```diff
@@ -135,7 +135,7 @@ func runCrossbuild() {
 			dockerPowerPCBuilderImage, "-i", repoPath, "-p", q(powerPCPlatformsParam))
 	}
 
-	if MIPSPlatformsParam := strings.Join(MIPSPlatforms[:], " "); MIPSPlatformsParam != "" {
+	if MIPSPlatformsParam := strings.Join(MIPSPlatforms[:], " "); MIPSPlatformsParam != "" && version.Compare(goVersion, "1.6", ">=") {
```

Comment:
We could, I also wanted to warn people when they specify a unknown/unhandled platform, but I didn't find a proper way to do it along side the platforms' matching.